### ISSUE TITLE
Possibl accessibility: No way to review finished block from keyboard with screen reader

### ISSUE THREAD

**Issue by zersiax** [marktext/marktext#2797]
### Description

When typing and hitting a newline, the text previously entered can seemingly no longer be focused using arrow keys. in any case, screen readers appear to be unable to regain focus on the previously entered text using regular navigation commands (up and down arrow keys). I could just be missing something, though?

- [x] Can you reproduce the issue? 

### Steps to reproduce

1. With a screen reader running, Type a heading and hit enter , twice just in case.
2. Try sing up arrow to go back, e.g. to edit the earlier entered heading.
3. Observe that the focus does not eturn to the previously entered content according to the screen reader.

**Expected behavior:**

Normally, you'd be able to go bck to, for example, fi a typo. 
**Actual behavior:**

You can't, you can review stuff as long as you don't hit the enter key, but after that it appears to no longer be doable.

### Versions

- MarkText version: 0.16.3
- Operating system: Windows 11

**Comment by Jocs** [comment]
@zersiax I can not reproduce this issue, please add a video related to this issue.

**Comment by Jocs** [comment]
We asked for more information to reproduce this issue but have not received a response. Closing due to inactivity — please reopen with reproduction steps if you can still reproduce this on the latest version.
